main.go

```diff
@@ -57,27 +57,43 @@ func process(typeName string, fileName string, packageName string, mode string)
 
 	var specs [][2]string
 
-	ast.Inspect(f, func(node ast.Node) bool {
-		spec, ok := node.(*ast.ValueSpec)
-		if !ok {
+	ast.Inspect(f, func(astNode ast.Node) bool {
+		node, ok := astNode.(*ast.GenDecl)
+		if !ok || node.Tok != token.CONST {
```

Comment:
will this work with?

```go
var (

....
)
```

Reply:
Good point. I wrong assumed `var` would not be used for enums.
Fixed.